src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailedPackageMetadata.cs

```diff
@@ -32,11 +32,19 @@ public DetailedPackageMetadata(IPackageSearchMetadata serverData, PackageDepreca
             Tags = serverData.Tags;
             DownloadCount = downloadCount;
             Published = serverData.Published;
-            DependencySets = serverData.DependencySets?
-                .Select(e => new PackageDependencySetMetadata(e))
-                ?? new PackageDependencySetMetadata[] { };
-            HasDependencies = DependencySets.Any(
-                dependencySet => dependencySet.Dependencies != null && dependencySet.Dependencies.Count > 0);
+
+            IEnumerable<PackageDependencyGroup> dependencySets = serverData.DependencySets;
+            if (dependencySets != null && dependencySets.Any())
+            {
+                DependencySets = dependencySets.OrderBy((ds) => ds.TargetFramework.DotNetFrameworkName)
+                    .OrderByDescending((ds) => ds.TargetFramework.Version)
```

Comment:
The `OrderBy(…).OrderByDescending(…)` seems unrelated to accessibility fixes.  It seems like it should be a separate issue/PR.

Reply:
Will remove this for now... ".OrderBy((ds) => ds.TargetFramework.DotNetFrameworkName)
                    .OrderByDescending((ds) => ds.TargetFramework.Version)" and merge into dev seperately.